src/lib/logger/impl.ml

```diff
@@ -303,18 +303,14 @@ module Consumer_registry = struct
             () )
 end
 
+[%%versioned
 module Stable = struct
   module V1 = struct
-    module T = struct
-      type t = {null: bool; metadata: Metadata.Stable.V1.t; id: string}
-      [@@deriving bin_io, version]
```

Comment:
RE what Avery mentioned: I would be surprised if this bin_io is being used at all. I can't think of any reason we need to serialize this.

Reply:
@nholland94 @Schmavery The serialized type is used in the `Worker.init_arg` types for `Prover` and `Verifier`, I believe.

Reply:
Hmmm I don't know if that's the way we should be doing that. I'll consider changing this in a separate PR.